### PR TITLE
my location lol

### LINKED ISSUE #3976: Bing wants to join
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by bingsu**
I marked my location @githubteacher.
👍 

 closes #3976

### CODE REVIEW

File: _pins/bing.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: bing
+Your Latitude: 34.0486
+Your Longitude: -118.4424
+---
```
Comment (hectorsector) [inline _pins/bing.json]: @bingsu Great work so far! You'll need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

Comment (neverlosemypassion) [inline _pins/bing.json]: I make the same mistake the first time.

Comment (neverlosemypassion) [inline _pins/bing.json]: I make the same mistake at the first time.

**Comment by brianamarie** [comment]
Hi @bingsu! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
@bingsu 🤔 I'm not sure why this test is failing. I will keep looking into it!

**Comment by brianamarie** [comment]
Woohoo! @bingsu I'm going to merge now. Nice work!

**Pull request merged**